code/__DEFINES/~skyrat_defines/jobs.dm

```diff
@@ -5,5 +5,5 @@
 #define JOB_UNAVAILABLE_LANGUAGE JOB_UNAVAILABLE_SPECIES + 1
 
 #define SEC_RESTRICTED_QUIRKS "Blind" = TRUE, "Brain Tumor" = TRUE, "Deaf" = TRUE, "Paraplegic" = TRUE, "Mute" = TRUE, "Foreigner" = TRUE, "Pacifist" = TRUE, "Chunky Fingers" = TRUE
-#define HEAD_RESTRICTED_QUIRKS "Blind" = TRUE, "Deaf" = TRUE, "Mute" = TRUE, "Foreigner" = TRUE, "Chunky Fingers" = TRUE
-#define TECH_RESTRICTED_QUIRKS "Chunky Fingers" = TRUE
+#define HEAD_RESTRICTED_QUIRKS "Deaf" = TRUE, "Mute" = TRUE, "Foreigner" = TRUE, "Chunky Fingers" = TRUE // SKYRAT EDIT
+#define TECH_RESTRICTED_QUIRKS "Chunky Fingers" = TRUE, "Blind" = TRUE // SKYRAT EDIT
```

Comment:
So you're making all engineers and atmos techs unable to be blind?
I don't necessarily disagree, but it's outside of the scope of your current PR.

Reply:
All in all, what this does is that it allows the RD, the QM (which might already be able to regardless), the HoP and the Captain and the NT Rep to be blind.

NT Rep is supposed to watch over Command, can't exactly "watch" when you're blind. HoP needs to be able to see people to properly man their line. I disagree with a blind Captain, I feel like it's just going to cripple Command too much. QM, I couldn't care less, but you can't do paperwork whilst blind, I'm pretty sure. RD, well, science experiments require you to be able to see things, you also can't do xenobio (or at least, shouldn't be able to), since you kinda need to look into the slime management console.

I'm really not convinced by this one, chief.

Reply:
> adding it to tech was required for giving ce tech restricted flags without bloating the file too much

Reply:
> NT Rep is supposed to watch over Command, can't exactly "watch" when you're blind. HoP needs to be able to see people to properly man their line. I disagree with a blind Captain, I feel like it's just going to cripple Command too much. QM, I couldn't care less, but you can't do paperwork whilst blind, I'm pretty sure. RD, well, science experiments require you to be able to see things, you also can't do xenobio (or at least, shouldn't be able to), since you kinda need to look into the slime management console.

in regard to this, i don't see any particular reason why the hop and nt rep, at least, shouldn't be allowed to be blind. that kind of thing prevents a large part of the potential workforce from doing their work, and from an ooc standpoint, just seems needlessly constraining. if someone wants to make it harder on themselves for a reason, and their job allows for full functionality despite that, i see no reason why it shouldn't be at least properly entertained

Reply:
HoP's whole job is paperwork, mechanically, sure, you can do paperwork blind. But ICly that doesn't make any sense.